libsupport/include/katana/CommBackend.h

```diff
@@ -37,6 +37,8 @@ class KATANA_EXPORT CommBackend {
   uint32_t Num{1};
   /// The id number of this task
   uint32_t ID{0};
+  /// The local id of this task (within its machine)
+  uint32_t Local;
```

Comment:
Could we make this something like `pid`? Or maybe just change the comment to say "The local pid of this task" assuming thats' what it is.

Reply:
We can change it, but it's not a `pid` in the OS sense. @roshandathathri suggested to call it `LocalRank` here: https://github.com/KatanaGraph/katana-enterprise/pull/2039#discussion_r713393936 . It's the MPI process ordinal number within its machine, used for determining the GPU device ID this process will use on a machine with multiple GPU devices.

Reply:
Changed to `Rank` and `LocalRank`